helm-chart/renku/templates/_helpers.tpl

```diff
@@ -65,10 +65,6 @@ Define subcharts full names
 {{- printf "%s-%s" .Release.Name "uiserver" | replace "+" "_" | trunc 63 | trimSuffix "-" -}}
 {{- end -}}
 
-{{- define "jupyterhub.fullname" -}}
```

Comment:
shouldn't this change be part of the official charts instead of the documentation?

Reply:
@pameladelgado ideally yes. But I forgot to do it in some of this other PRs and saw it when going through the docs. I can open a new PR just for this change if you wish. Leaving or removing this helper function makes no difference to whether the deployment works or not. Let know if you want me to move this to another PR. I think it is ok to leave it here but I am happy to move it to another PR if you think otherwise.

Reply:
OK I removed this change and will open a new PR just for this.